Watchman.Discord/Areas/Responses/Services/ResponsesGetterService.cs

```diff
@@ -25,14 +29,39 @@ public IEnumerable<Response> GetResponsesFromBase()
 
         public IEnumerable<Response> GetResponsesFromResources()
         {
+            Log.Information("Initializing default responses...");
+            var managerMethods = typeof(ResponsesManager).GetMethods();
             var defaultResponses = typeof(Devscord.DiscordFramework.Framework.Commands.Responses.Resources.Responses).GetProperties()
                 .Where(x => x.PropertyType.Name == "String")
                 .Select(prop =>
```

Comment:
spory ten select
jak robi sie taki duży to lepiej rozdzielić na osobne metody

Reply:
usunąłem switcha, teraz jest trochę mniejszy, poza tym to nie powinien być problem